[Docu 2405] How to publish an API spec on Dev Portal (#4266)
Kong/docs.konghq.com: src/gateway/kong-enterprise/developer-portal/publish-spec.md

---
title: Publish an API Spec to the Dev Portal
badge: enterprise
content_type: how-to
---

> The Dev Portal provides a single source of truth for all developers to locate, access, and consume services. With intuitive content management for documentation, streamlined developer onboarding, and role-based access control (RBAC), the Dev Portal provides a comprehensive solution for creating and customizing a unified developer experience.

Author: ```suggestion
Dev Portal provides a single source of truth for all developers to locate, access, and consume services. With intuitive content management for documentation, streamlined developer onboarding, and role-based access control (RBAC), the Dev Portal provides a comprehensive solution for creating and customizing a unified developer experience.
```

---

> title: How to Publish an API Spec to the Dev Portal

Reviewer: ```suggestion
title: Publish an API Spec to the Dev Portal
```

badge: enterprise
content_type: how-to
---

> Dev Portal provides a single source of truth for all developers to locate, access, and consume services. With intuitive content management for documentation, streamlined developer onboarding, and role-based access control (RBAC), the Dev Portal provides a comprehensive solution for creating and customizing a unified developer experience.

Reviewer: ```suggestion
The Dev Portal provides a single source of truth for all developers to locate, access, and consume services. With intuitive content management for documentation, streamlined developer onboarding, and role-based access control (RBAC), the Dev Portal provides a comprehensive solution for creating and customizing a unified developer experience.
```

> ## Publish a Spec to Dev Portal

Reviewer: ```suggestion
```
I think you might not need this heading since it would just be repeating the title, but I'm not sure what the precedent is here about omitting h2 before a task

> 1. In Kong Manager, navigate to **Dev Portal** > **Editor** and open the link in a new tab. The editor lets you customize the Dev Portal.

Reviewer: ```suggestion
1. In Kong Manager, navigate to **Dev Portal** > **Editor** and open the link in a new tab.
```
Just my opinion, I didn't think the last sentence was necessary for completing the task

> 2. Click on **New File** to add a new spec.

Reviewer: ```suggestion
2. Click **New File** to add a new spec.
```

> 3. Open the dropdown, select **spec**.

Reviewer: ```suggestion
3. Select **spec** from the drop-down menu.
```

4. Click **Create File**.

6. Paste the contents of your API spec.

7. Click **Save Changes**.

> ## More Information

Reviewer: ```suggestion
## More information
```

> * [Customizing the look and feel of the site and editor](/gateway/{{page.kong_version}}/developer-portal/theme-customization/easy-theme-editing/)

Reviewer: ```suggestion
* [Customizing the look and feel of the site and editor](/gateway/{{page.kong_version}}/kong-enterprise/developer-portal/customize/easy-theme-editing/)
```
The original link was a 404, so I think this is the right link?

> * [Managing access](/gateway/{{page.kong_version}}/developer-portal/administration/managing-developers/)

Reviewer: ```suggestion
* [Managing access](/gateway/{{page.kong_version}}/kong-enterprise/developer-portal/authentication/managing-developers/)
```
The original link was a 404, so I think this is the right link?

> * [Configuring the Dev Portal](/gateway/{{page.kong_version}}/developer-portal/configuration/workspaces/)

Reviewer: This link is a 404, not sure what we replaced it with in the new IA nav
